Makefile

```diff
@@ -11,21 +11,18 @@ TARGET := $(shell lscpu | head -1 | sed -e 's/Architecture:\s*//g')
 COVERAGE_DIR ?= /tmp/cov
 GEF_PATH ?= $(shell readlink -f gef.py)
 TMPDIR ?= /tmp
-PYTEST_PARAMETERS := --verbose -n $(NB_CORES)
+PYTEST_PARAMETERS := --verbose --forked --numprocesses=$(NB_CORES)
 ifdef DEBUG
 	PYTEST_PARAMETERS += --pdb
```

Comment:
As far as I understand `--pdf` only works on `pytest` when using a single process.
We should either remove this `DEBUG` block or set `numprocesses` to `0` if we do use the flag.

Reply:
I'd say we remove it then, so we use `make test` for the full test, then go granular for each failure with `pytest` directly on the command line.
Opinion?

Reply:
fine with me! But maybe we document it somewhere as well, so future contributors can find this in our docs?

Reply:
Something along the line: "If a tests fails it can directly be debugged using..."

Reply:
Excellent point! We don't even have a page about testing. I'll add one to the PR